apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/PatientClaimsEobTaskTransformerV2.java

```diff
@@ -323,21 +324,19 @@ private <T> List<T> findClaimTypeByPatient() {
     criteria.where(wherePredicate);
 
     List<T> claimEntities = null;
-    Long eobsByBeneIdQueryNanoSeconds = null;
+    long eobsByBeneIdQueryNanoSeconds;
```

Comment:
It's nice that its not boxed any more but it should also be a local within the finally block since that's the only place it's used.

Reply:
Moved them all into scope where they occur.